python/adelphi/adelphi/gh.py

```diff
@@ -22,7 +22,6 @@
 
 from github import Github
 
-logging.basicConfig(level=logging.INFO)
```

Comment:
This (and store.py below) should definitely not be setting up their own logging... they should be leveraging logs that are configured somewhere else.